src/bloggy_frontend/src/context/AuthContext.jsx

```diff
@@ -9,22 +9,33 @@ export const AuthProvider = ({ children }) => {
   const [isConnecting, setIsConnecting] = useState(false)
   const [error, setError] = useState(null)
 
-  const connectWallet = async () => {
-    setIsConnecting(true)
-    setError(null)
-    try {
-      const artemisWalletAdapter = new Artemis()
-      await artemisWalletAdapter.connect('plug')
-      console.log('Artemis wallet connected:', artemisWalletAdapter)
-      setArtemisAdapter(artemisWalletAdapter)
-      setIsConnected(true)
-    } catch (error) {
-      console.error('Error connecting Artemis wallet:', error)
-      setError('Failed to connect to the wallet. Please try again.')
-    } finally {
-      setIsConnecting(false)
+const connectWallet = async () => {
+  setIsConnecting(true);
+  setError(null);
+  try {
+    if (typeof window.ic === 'undefined' || typeof window.ic.plug === 'undefined') {
+      setError('Plug Wallet is not installed. Please install the Plug Wallet extension.');
```

Comment:
This is fine but just as an FYI artemis has this inbuild example 
```js
artemis.wallets.filter(wallet => wallet.id === "plug")[0].adapter.readyState 
```
this returns a string is installed `"Installed"`, if not returns `"NotDetected" `

Reply:
Sure, I will make the changes